.nova/Configuration.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should this file be .gitignored?

Reply:
That was an accidental commit by me, I'll remove it!

Reply:
Removed!